src/main/java/seedu/address/logic/commands/RemarkCommand.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.Remark;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Edits the remark of a person to the address book.
+ */
+public class RemarkCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "remark";
+    public static final String COMMAND_ALIAS = "rm";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
mention that the remark command also deletes the remark when you do a `remark r/`

Reply:
`: If Remarks the person identified by the index number used in the last person listing.\n`
`  If the remark field is empty, the remark is removed for the person.`

Reply:
` "Example: (add remark)" + COMMAND_WORD + " 1" + PREFIX_REMARK + "Likes to drink coffee." `
` "Example: (delete remark)" + COMMAND_WORD + " 1" + PREFIX_REMARK" `

Reply:
ok i'll check it